src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierCalculationService.kt

```diff
@@ -80,15 +80,15 @@ class TierCalculationService(
   private fun getLatestTierCalculation(crn: String): TierCalculationEntity? =
     tierCalculationRepository.findFirstByCrnOrderByCreatedDesc(crn).also {
       when (it) {
-        null -> log.info("No tier calculation found for $crn")
+        null -> log.warn("No tier calculation found for $crn")
```

Comment:
warn?
This will always happen for new offenders

Reply:
I think this and the one below is but I've just realised the logging is on the private method that is called from multiple places. I'll  move it to the public one I assumed it was in, if they're calling our service for a CRN that we haven't calculated we ought to find out why because we should have calculated it.